labels.mss

```diff
@@ -78,8 +78,11 @@
   shield-text-dx: 2;
   shield-unlock-image: true;
   shield-min-distance: 10;
-  [ldir!=''] {
+  #city[ldir!=''] {
```

Comment:
I suggest two things:
- use a null value (by coercing in SQL: `NULLIF(ldir, '') as ldir`)
- also emit a `ldir` column for countries, so the final XML will be simpler (CartoCSS to XML can become quickly a mess when using partial subrules), no need for that to add it in the CSV, just create a dummy value in SQL `SELECT … NULL AS ldir`

Also, when you change the SQL, make sure you change it in both styles if needed (at lot of layers are duplicated for now) ;)

Note: I may work on a better factorization between the two styles Layers at some point, but given this is not a Cartocss feature, neither a Kosmtik feature and neither a YML feature, I preferred for now to see how we are going before investing time in developing a "include" feature in Kosmtik. :)

Reply:
I think I made the necessary changes, tell me if not

Thanks for the suggestions and feedbacks